src/app/my-account/my-account/my-account.component.ts

```diff
@@ -44,7 +53,34 @@ export class MyAccountComponent implements OnInit {
     dialogRef.componentInstance.onClosed.pipe(take(1)).subscribe((confirmed: boolean) => {
       dialogRef.close();
       if (confirmed) {
-        // TODO: call to api
+        if (announcement.type === TransportOffer.TypeEnum.Transport) {
+          this.transportResourceService
+            .deleteTransport(announcement.id!)
```

Comment:
This `!` looks dangerous. Are we sure all offers have an ID? If that's the case then please report to the backend that they need to make this field required and we'll regenerate API then.